compiler/rustc_errors/src/lib.rs

```diff
@@ -1005,10 +1004,9 @@ impl Handler {
         self.inner.borrow_mut().delay_span_bug(span, msg)
     }
 
-    // FIXME(eddyb) note the comment inside `impl Drop for HandlerInner`, that's
-    // where the explanation of what "good path" is (also, it should be renamed).
-    pub fn delay_good_path_bug(&self, msg: impl Into<DiagnosticMessage>) {
-        self.inner.borrow_mut().delay_good_path_bug(msg)
+    // FIXME(eddyb) note the comment inside `impl Drop for HandlerInner`.
```

Comment:
I don't know what this FIXME is actually asking for... to copy the explanation and give this function a good doc comment? I'm gonna leave it for now lol.